wagtailmenus/menuadmin.py

```diff
@@ -87,9 +87,14 @@ class MainMenuAdmin(SnippetViewSet):
     index_view_class = MainMenuIndexView
 
     edit_view_class = MainMenuEditView
-    edit_template_name = "wagtailmenus/mainmenu_edit.html"
+    if WAGTAIL_VERSION < (6, 3):
```

Comment:
**Question**: Shouldn't we do the opposite and make the wagtail_63/mainmenu_edit.html the canonical version?

Reply:
@MrCordeiro You're right, I'll switch it around as soon as I can.